### ISSUE TITLE
Fixed Card layout: Cards within the screen are not clearly visible, when the application is resized to 320 CSS pixels.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3815]
Description: Cards within the screen are not clearly visible, when the application is resized to 320 CSS pixels.

Expected: Ideally, all the UI elements should be clearly visible to the user, when screen is resized to 320 CSS pixels.

Screenshot: 

![image](https://user-images.githubusercontent.com/32538291/98772763-7faab200-240d-11eb-89d6-eef69071c6c0.png)

**Comment by DeepakSap14** [comment]
Minimum width of card as per fiori spec should be 20rem(320px). but in example it is under core "documentation area" which is also taking space. that is why scroll is happening here.
